code/modules/core_implant/cruciform/rituals/construction.dm

```diff
@@ -607,7 +607,7 @@ GLOBAL_LIST_INIT(nt_constructs, init_nt_constructs())
 	name = "Absolutism Medkit"
```

Comment:
Could you actually comment out this recipe? It's a good idea, but I haven't figured out how to stop our resident powergamers from printing it with the printer and then Uprooting it to get free mats